libmaven/mzMassSlicer.cpp

```diff
@@ -44,16 +45,20 @@ void MassSlices::algorithmB(float userPPM,int rtStep) {
 	this->_precursorPPM=userPPM;
 
     int totalScans = 0,currentScans = 0;
+
+//Calculate the total number of scans
     for(unsigned int i=0; i < samples.size(); i++) totalScans += samples[i]->scans.size();
 
+//Calculating the rt window using average distance between rts and mutiplying it with 20
     if (samples.size() > 0 and rtStep > 0) rtWindow = (samples[0]->getAverageFullScanTime()*rtStep);
 
     sendSignal("Status", 0 , 1);
 
-// #pragma omp parallel for ordered
+// Looping over every sample
     for(unsigned int i=0; i < samples.size(); i++) {
         if (slices.size() > _maxSlices) break;
 
+// Checking if splicing has been stopped
```

Comment:
@Raghavdata This is to stop the peak detection. Description that is given is not accurate.